Source/AliveLibAE/Renderer/GLShader.cpp

```diff
@@ -0,0 +1,208 @@
+#include "GLShader.hpp"
+
+GLShader::GLShader()
+{
+    
+}
+
+void printProgramLog(GLuint program)
+{
+    //Make sure name is shader
+    if (glIsProgram(program))
+    {
+        //Program log length
+        int infoLogLength = 0;
+        int maxLength = infoLogLength;
+
+        //Get info string length
+        glGetProgramiv(program, GL_INFO_LOG_LENGTH, &maxLength);
```

Comment:
you should probably have a function or macro to check for errors - all these gl apis can fail